src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -63,7 +64,12 @@ def wait_process(proc, wait_time):
     @staticmethod
     def cleanup_processes():
         """ Clean up cart processes, in case avocado/apricot does not. """
-        stop_processes(["localhost"], "'(crt_launch|orterun)'")
+        error_list = []
+        localhost = socket.gethostname().split(".")[0]
```

Comment:
We need `localhost` to be a `list` object:
```suggestion
        localhost = socket.gethostname().split(".")[0:1]
```